tests/unit_tests/data_model/test_meta_data_model.py

```diff
@@ -0,0 +1,79 @@
+import copy
+
+import pytest
+
+from simtools.data_model import meta_data_model
+
+
+def test_top_level_reference_schema():
+    """(very hard to test this)"""
```

Comment:
What about testing also that the output is what you expect? For example that you have at least one key you are expecting and in this case also a nested dict?

Reply:
I've added an `assert` for a random key to be included. Actually, the test is not so bad and I removed the 'very hard to test comments'. We test three characteristics of the outcome of this module, that should be good!